scenarios/openshift/pod_network_outage.yml

```diff
@@ -0,0 +1,16 @@
+# yaml-language-server: $schema=../plugin.schema.json
+- id: pod_network_outage
+  config:
+    namespace: <namespace>              # Required - Namespace of the pod to which filter need to be applied
+    direction:                          # Optioinal - List of directions to apply filters
+        - <egress/ingress>              # Default both egress and ingress
+    ingress_ports:                      # Optional - List of ports to block traffic on
+        - <port number>                 # Default [], i.e. all ports
+    egress_ports:                       # Optional - List of ports to block traffic on
+        - <port number>                 # Default [], i.e. all ports
+    pod_name: <pod name>                # When label_selector is not specified, pod matching the name will be selected for the chaos scenario
+    label_selector: <label_selector>    # When pod_name is not specified, pod with matching label_selector is selected for chaos scenario
+    instance_count: <number>            # Number of nodes to perform action/select that match the label selector
+    wait_duration: <time_duration>      # Default is 300. Ensure that it is at least about twice of test_duration
+    test_duration: <time_duration>      # Default is 120
+    kraken_config: <path>               # Specify this if you want to use Cerberus config
```

Comment:
What is this variable doing? I'm not sure what to set here. The variable says kraken but comment says cerberus?

Reply:
kraken_config is the path to kraken config file, to publish kraken status to cerberus and also to get go no-go signal from cerberus

Reply:
@yogananth-subramanian and I discussed leveraging the required params from kraken config and try removing this parameter.

Reply:
@paigerube14 @chaitanyaenr removed the requirement to explicitly pass kraken config file in commit https://github.com/redhat-chaos/krkn/commit/ff1a489cd6cca3e47c5967b849d1189de0e591aa. The plugin automatically passes the kraken config file , just like kubeconfig file.